variables.tf

```diff
@@ -76,31 +76,31 @@ variable "server_boot_image" {
 
 variable "server_instance_metadata" {
   description = "Metadata key/value pairs to make available from within the server instance."
-  type        = "map"
+  type        = map(string)
   default     = {}
 }
 
 variable "server_grpc_allow_ranges" {
   description = "List of CIDRs that will be allowed gRPC access to forseti server"
-  type        = "list"
+  type        = list(string)
   default     = ["10.128.0.0/9"]
 }
 
 variable "server_ssh_allow_ranges" {
   description = "List of CIDRs that will be allowed ssh access to forseti server"
-  type        = "list"
+  type        = list(string)
   default     = ["0.0.0.0/0"]
 }
 
 variable "server_tags" {
   description = "GCE Forseti Server VM Tags"
-  type        = "list"
+  type        = list(string)
   default     = []
 }
 
 variable "server_access_config" {
   description = "Server instance 'access_config' block"
-  type        = "map"
+  type        = map(string)
```

Comment:
This should be an object since there is a static set of expected keys.

Reply:
Fixed